mqtt/mqtt-lwt/src/test/resources/qdrouterd.conf

```diff
@@ -50,8 +50,9 @@ listener {
 # listener for services (i.e. Last Will and Testament and Subscription services)
 listener {
   host: 0.0.0.0
-  port: 55673
-  authenticatePeer: no
```

Comment:
Without the authenticatePeer, you don't need to use a SASL layer. I recognise this is only a test though, so perhaps not significant here.

Reply:
I can add it for completeness, though I think we want to refactor the test to run without the routet